laravel5/app/Monomalpoly/Partie.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include('Case.php');
+
+class Partie{
+	private $listeCouleur = ['red','yellow','orange','blue','green','grey'];
+	private $listeNomCase = [];
```

Comment:
Probably bad naming. It makes me believe "game boxes" have a name and that there's a list for it.